app/locale/de-DE.ts

```diff
@@ -92,6 +92,8 @@ export default {
   unlockQueue: 'Warteschlange öffnen',
   username: 'Benutzername',
   users: 'Benutzer',
+  maxUsers: 'Max users', // need translate
+  unlimitedUsers: 'Unlimited users', // need translate
```

Comment:
You can remove these from the other languages, they'll fallback to the English strings.